modules/custom_operations/user_ie_extensions/tokenizer/python/README.md

```diff
@@ -278,302 +280,309 @@ This report is autogenerated and includes tokenizers and detokenizers tests. The
     </tr>
     <tr>
       <td >SentencePiece</td>
       <td >codellama/CodeLlama-7b-hf_slow</td>
       <td >100.00</td>
-      <td >105</td>
+      <td >180</td>
     </tr>
     <tr>
       <td >SentencePiece</td>
       <td >microsoft/deberta-v3-base</td>
-      <td >88.57</td>
-      <td >105</td>
+      <td >93.33</td>
+      <td >180</td>
     </tr>
     <tr>
       <td >SentencePiece</td>
       <td >microsoft/deberta-v3-base_slow</td>
-      <td >96.19</td>
-      <td >105</td>
+      <td >100.00</td>
+      <td >180</td>
     </tr>
     <tr>
       <td >SentencePiece</td>
       <td >xlm-roberta-base</td>
-      <td >95.24</td>
-      <td >105</td>
+      <td >98.89</td>
+      <td >180</td>
     </tr>
     <tr>
       <td >SentencePiece</td>
       <td >xlm-roberta-base_slow</td>
-      <td >95.24</td>
-      <td >105</td>
+      <td >98.89</td>
+      <td >180</td>
     </tr>
     <tr>
       <td >SentencePiece</td>
       <td >xlnet-base-cased</td>
-      <td >46.67</td>
-      <td >105</td>
+      <td >61.11</td>
+      <td >180</td>
     </tr>
     <tr>
       <td >SentencePiece</td>
       <td >xlnet-base-cased_slow</td>
-      <td >40.00</td>
-      <td >105</td>
+      <td >53.33</td>
+      <td >180</td>
     </tr>
     <tr>
       <td >Tiktoken</td>
       <td >Qwen/Qwen-14B-Chat</td>
-      <td >98.10</td>
-      <td >105</td>
+      <td >98.15</td>
+      <td >108</td>
     </tr>
     <tr>
       <td >Tiktoken</td>
       <td >Salesforce/xgen-7b-8k-base</td>
-      <td >97.14</td>
-      <td >105</td>
+      <td >97.22</td>
+      <td >108</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >ProsusAI/finbert</td>
-      <td >84.62</td>
-      <td >39</td>
+      <td >80.00</td>
+      <td >40</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >bert-base-multilingual-cased</td>
-      <td >84.62</td>
-      <td >39</td>
+      <td >80.00</td>
+      <td >40</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >bert-large-cased</td>
-      <td >84.62</td>
-      <td >39</td>
+      <td >80.00</td>
+      <td >40</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >cointegrated/rubert-tiny2</td>
-      <td >84.62</td>
-      <td >39</td>
+      <td >80.00</td>
+      <td >40</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >distilbert-base-uncased-finetuned-sst-2-english</td>
-      <td >84.62</td>
-      <td >39</td>
+      <td >80.00</td>
+      <td >40</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >google/electra-base-discriminator</td>
-      <td >84.62</td>
-      <td >39</td>
+      <td >80.00</td>
+      <td >40</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >google/mobilebert-uncased</td>
-      <td >100.00</td>
-      <td >39</td>
+      <td >95.00</td>
+      <td >40</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >jhgan/ko-sbert-sts</td>
-      <td >79.49</td>
-      <td >39</td>
+      <td >75.00</td>
```

Comment:
`WordPiece` became worse, why?

Reply:
I added a new test case with a control char that the current regex normalizer does not cover.